bandit/plugins/trojansource.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding:utf-8 -*-
+#
+# SPDX-License-Identifier: Apache-2.0
+
+r"""
+=====================================================
+B113: TrojanSource - Bidirectional control characters
+=====================================================
+
+This plugin checks for the presence of unicode bidirectional control characters
+in Python source files. Those characters can be embedded in comments and strings
+to reorder source code characters in a way that changes its logic.
+
+:Example:
+
+.. code-block:: none
+
+    >> Issue: [B113:trojansource] A Python source file contains bidirectional control characters ('\u202e').
+       Severity: High   Confidence: Medium
+       Location: examples/trojansource.py:0:0
+
+.. seealso::
+
+ .. [1] https://trojansource.codes/
+ .. [2] https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42574
+
+.. versionadded:: 1.7.2
+
+"""  # noqa: E501
+
+from tokenize import detect_encoding
+
+import bandit
+from bandit.core import test_properties as test
+
+
+BIDI_CHARACTERS = ('\u202A', '\u202B', '\u202C', '\u202D', '\u202E',
+                   '\u2066', '\u2067', '\u2068', '\u2069')
+
+
+@test.test_id('B113')
```

Comment:
PR https://github.com/PyCQA/bandit/pull/743 is also using B113.  First to merge would get it, other would need to change.

Reply:
Ok, I'm fine with that 😄

Reply:
Consider adding `\u200F` (RIGHT-TO-LEFT MARK) as well.
That's the one used in https://www.python.org/dev/peps/pep-0672/ for their example.